src/QMCWaveFunctions/TWFFastDerivWrapper.h

```diff
@@ -14,6 +15,12 @@
 #define QMCPLUSPLUS_TWFFASTDERIVWRAPPER_H
 
 #include "QMCWaveFunctions/WaveFunctionComponent.h"
+#include "ResourceCollection.h"
+#include "Utilities/ResourceCollection.h"
+#include "QMCWaveFunctions/SPOSet.h"
+#include "OMPTarget/ompBLAS.hpp"
+#include "Numerics/MatrixOperators.h"
+#include <AccelBLAS.hpp>
```

Comment:
All the BLAS related header include can be moved to the cpp file with TWFFastDerivWrapperMultiWalkerMem